plugins/com.python.pydev.analysis/plugin.xml

```diff
@@ -177,7 +177,7 @@
                menubarPath="org.python.pydev.ui.actions.menu/pydev"
                tooltip="Force a code analysis for the selected elements (recursive)">
          </action>
-         <visibility><objectState name="projectNature" value="org.python.pydev.pythonNature"/></visibility>
+         <visibility><objectState name="contentTypeId" value="org.python.pydev.pythonfile"/></visibility>
```

Comment:
I had that same thought shortly after putting up the PR. I'll see wether that can be expressed, don't have too much experience with this kind of object contribution, and otherwise will discard the request.